src/pages/public_registration/public_files/preview_file/PreviewPdf.tsx

```diff
@@ -4,15 +4,23 @@ import { dataTestId } from '../../../../utils/dataTestIds';
 import { CommonPreviewProps } from './PreviewFile';
 import { PreviewUnavailable } from './PreviewUnavailable';
 
+const browserDoesNotSupportingOnLoad = (): boolean => {
+  return isSafariBrowser();
+};
+
+const isSafariBrowser = (): boolean => {
+  return navigator.userAgent.toLowerCase().indexOf('safari/') > -1;
+};
+
 export const PreviewPdf = ({ url, altText, ...props }: CommonPreviewProps) => {
-  const [successfullyLoadedPdf, setSuccessfullyLoadedPdf] = useState(false);
+  const [successfullyLoadedPdf, setSuccessfullyLoadedPdf] = useState(browserDoesNotSupportingOnLoad());
```

Comment:
Ser ikke ut som safari sender noen event på onLoad, så en liten hakk for å få vist frem til safari brukere.

Reply:
Ah, så ikke det da jeg bruker Firefox